content/en/docs/concepts/security/index.md

```diff
@@ -12,565 +12,543 @@ aliases:
+Breaking down a monolithic application into atomic services offers various
+benefits, including better agility, better scalability and better ability to
+reuse services. However, microservices also have particular security needs:
+
+- To defend against man-in-the-middle attacks, they need traffic encryption.
+- To provide flexible service access control, they need mutual TLS and
+   fine-grained access policies.
+- To determine who did what at what time, they need auditing tools.
+
+Istio Security provides a comprehensive security solution to solve these issues.
+This page gives an overview on how you can use Istio security features to secure
+your services, wherever you run them. In particular, Istio security mitigates
+both insider and external threats against your data, endpoints, communication,
+and platform.
+
+{{< image width="75%"
+    link="./overview.svg"
+    caption="Security overview"
+    >}}
 
-- **Defense in depth**: integrate with existing security systems to provide multiple layers of defense
+The Istio security features provide strong identity, powerful policy,
+transparent TLS encryption, and authentication, authorization and audit (AAA)
+tools to protect your services and data. The goals of Istio security are:
 
-- **Zero-trust network**: build security solutions on untrusted networks
+- Security by default: no changes needed to application code and
+    infrastructure
+- Defense in depth: integrate with existing security systems to provide
+    multiple layers of defense
+- Zero-trust network: build security solutions on distrusted networks
 
-Visit our [Mutual TLS Migration docs](/docs/tasks/security/authentication/mtls-migration/) to start using Istio security features with your deployed services.
-Visit our [Security Tasks](/docs/tasks/security/) for detailed instructions to use the security features.
+Visit our
+[mutual TLS Migration docs](/docs/tasks/security/authentication/mtls-migration/)
+to start using Istio security features with your deployed services. Visit our
+[Security Tasks](/docs/tasks/security/) for detailed
+instructions to use the security features.
 
 ## High-level architecture
 
 Security in Istio involves multiple components:
 
-- **Citadel** for key and certificate management
+- A Certificate Authority (CA) for key and certificate management
+- Istiod distributes to the proxies:
 
-- **Sidecar and perimeter proxies** to implement secure communication between clients and servers
+    - [authentication policies](/docs/concepts/security/#authentication-policies)
+    - [authorization policies](/docs/concepts/security/#authorization-policies)
+    - [secure naming information](/docs/concepts/security/#secure-naming)
 
-- **Pilot** to distribute [authentication policies](/docs/concepts/security/#authentication-policies)
-  and [secure naming information](/docs/concepts/security/#secure-naming) to the proxies
+- Sidecar and perimeter proxies work as Policy Enforcement Points (PEPs) to
```

Comment:
```suggestion
- Sidecar and perimeter proxies work as [Policy Enforcement Points](link to what is it) (PEPs) to
```

Reply:
@frankbu I could use the following link: https://www.jerichosystems.com/technology/glossaryterms/policy_enforcement_point.html
WDYT?